### PR TITLE
Tsunami hazard classes fix #1251

### LINKED ISSUE #1251: Update: Tsunami hazard zones - update metadata.py
BG has tsunami hazard zone descriptions in 2 languages on published maps.

Obtain definition text for tsunami hazard zones (english and Indonesian) from BG. & use these in metadata.py

#cc @dika

### PR DESCRIPTION
**Pull request by Charlotte-Morgan**
update tsunami_raster_hazard_classes
this includes english definitions for 3 classes
Indonesian definitions are available in #1251

### CODE REVIEW

**Comment by Charlotte-Morgan** [comment]
hi @ismailsunni @timlinux - please review
I can not see where this text is used - its not in the wizards or in the reports
A dancing princess with no audience?

File: safe/impact_functions/inundation/tsunami_population_evacuation_raster/metadata_definitions.py [hunk 1]
```diff
@@ -104,7 +104,15 @@ def as_dict():
             'limitations': [tr(
                 'The default threshold of 0.7 meter was selected based on '
                 'consensus, not hard evidence.')],
-            'citations': [],
+            'citations': [
+                tr('Papadopoulos, Gerassimos A., and Fumihiko Imamura. '
+                   '"A proposal for a new tsunami intensity scale." '
+                   'ITS 2001 proceedings, no. 5-1, pp. 569-577. 2001.'),
+                tr('Hamza Latief. pers com. Default impact threshold for tsunami'
```
Comment (ismailsunni) [inline safe/impact_functions/inundation/tsunami_population_evacuation_raster/metadata_definitions.py]: Missing space in the end.

**Comment by ismailsunni** [comment]
Hi @Charlotte-Morgan 

> hi @ismailsunni @timlinux - please review
I can not see where this text is used - its not in the wizards or in the reports
A dancing princess with no audience?

I checked it, there is no IF / code using it. All tsunami raster specific are using `meter` or `feet`. We can put it in generic raster. But, there will be a case where we assign non-tsunami hazard and there are option for `tsunami_raster_hazard_classes` (Old problem). Our InaSAFE / wizard hasn't been that smart to filter the classes based on the choice in the hazard type step.

Btw, there are some lines that more than 80 chars. Here the list:
```
./safe/definitions.py:694:80: E501 line too long (80 > 79 characters)
./safe/definitions.py:697:80: E501 line too long (80 > 79 characters)
./safe/definitions.py:709:80: E501 line too long (81 > 79 characters)
./safe/definitions.py:710:80: E501 line too long (80 > 79 characters)
./safe/definitions.py:997:80: E501 line too long (80 > 79 characters)
./safe/impact_functions/inundation/tsunami_population_evacuation_raster/metadata_definitions.py:111:80: E501 line too long (81 > 79 characters)
./safe/impact_functions/inundation/tsunami_population_evacuation_raster/metadata_definitions.py:112:80: E501 line too long (80 > 79 characters)
./safe/impact_reports/land_cover_report_mixin.py:98:80: E501 line too long (80 > 79 characters)
```

It's easy to fix, but I can help if you want.

**Comment by Charlotte-Morgan** [comment]
ok - so maybe it goes back to my comment in #1251 that I don't thing this is the right place to put this information :)
maybe @timlinux can advise where this type of information should go?

**Comment by ismailsunni** [comment]
In my opinion, we need to create new Tsunami IF from the current one, with some modification in the IF metadata, and classification. By doing this, we can put the tsunami_raster_classes in this "new" tsunami IF.

The difference from the current one:
1. The new IF will accept tsunami_classes, the current one will accept metres or feet
2. A little modification in the run method of the IF.

This reminds me that we need to have proper document / wiki to create new IF and document everything related IF (metadata, definitions.py, etc)

**Comment by Charlotte-Morgan** [comment]
hey @ismailsunni - i think I have either fixed it or broken it more.
Time for me to go home.
please merge if possible, if not; seems like it was irrelevant for now anyway and we can save it for another time.

**Pull request merged**